versions/1/elasticsearch/spec.yml

```diff
@@ -9,7 +9,7 @@ spec:
         - description: An ML model file
           type: file
           contentMediaType: "application/json"
-          pattern: '^{PACKAGE_NAME}-.+\.json$'
+          pattern: '^{PACKAGE_NAME}_.+\.json$'
```

Comment:
It looks like a breaking change for existing resources if we have already pushed anything to the integrations repo. Have we already done it? 

If so, then we need to modify the regexp to accept both entries.

Reply:
Cool, thanks for confirmation!